### ISSUE TITLE
Question: Any chance a more recent stdlib could be pulled in?

### ISSUE THREAD

**Issue by lubgr** [google/go-jsonnet#750]
I ran into a known issue with the `%g` format specifier, where `'%g' % 0` triggers an error. This has been [fixed](https://github.com/google/jsonnet/commit/e6bf0ee080fb0de6a6324bbc52346ceac750e5f3) a year ago, but since there was no upstream/lockstep version bump since then, the change has not yet made it into the submodule.

Could the submodule be updated to bring in a more recent stdlib, even if it is not a proper release? Or would it even make sense to bump the version?

Thanks for maintaining this repo by the way, it's a pleasure to work with it!

**Comment by sbarzowski** [comment]
Yeah, we can pull a newer stdlib. It is normally done as part of the release, but there's nothing stopping us from doing it separately. 

Feel free to submit a PR following instructions:
* https://github.com/google/go-jsonnet?tab=readme-ov-file#updating-and-modifying-the-standard-library
* https://github.com/google/go-jsonnet?tab=readme-ov-file#update-cpp-jsonnet-sub-repo

**Comment by johnbartholomew** [comment]
The standard library has now been updated in go-jsonnet master (as of Jan 18 - #782)

**Comment by lubgr** [comment]
Many thanks!
